modules/gapi/test/common/gapi_video_tests_inl.hpp

```diff
@@ -88,6 +89,69 @@ TEST_P(BuildPyr_CalcOptFlow_PipelineTest, AccuracyTest)
     compareOutputsOptFlow(outOCV, outGAPI);
 }
 
+#ifdef HAVE_OPENCV_VIDEO
+TEST_P(BackgroundSubtractorMOG2Test, AccuracyTest)
+{
+    initTestDataPath();
+
+    cv::gapi::video::BackgroundSubtractorParams bsp(cv::gapi::video::TYPE_BS_MOG2, 500, 16, true, -1);
```

Comment:
Maybe you should test more different configurations. You can make all of it test parameters

Reply:
Done